### PR TITLE
os/kernel/binary_manager : Change lldbg to printf

### PR DESCRIPTION
**Pull request by jeongchanKim**
To show always to user, it is better to use printf instead of lldbg.
So change lldbg to printf in binary manager.

Signed-off-by: jeongchanKim <jc_.kim@samsung.com>

### CODE REVIEW

**Review by davidfather — APPROVED** [review]
lgtm.

File: os/kernel/binary_manager/binary_manager_recovery.c [hunk 1]
```diff
@@ -92,7 +92,7 @@ static void binary_manager_reset_board(void)
 	(void)irqsave();
 	sched_lock();
 	for (;;) {
-		lldbg("\nASSERT!! Push the reset button!\n");
+		printf("\nASSERT!! Push the reset button!\n");
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager_recovery.c:95]: This is the assert code so that lldbg is correct. Let's revert this.

Comment (sunghan-chang) [reply os/kernel/binary_manager/binary_manager_recovery.c:95]: How about adding comment about the reason why here uses lldbg instead of printf?

Comment (jeongchanKim) [reply os/kernel/binary_manager/binary_manager_recovery.c]: Reverted and added the reason why changed to printf in the commit msg.
Thanks.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**